### PR TITLE
column edit to same length

### PR DESCRIPTION
**Pull request by madhabpaul**
Signed-off-by: madhabpaul <madhab12paul@gmail.com>

### CODE REVIEW

File: _data/categories/gw.yml [hunk 1]
```diff
@@ -12,4 +12,22 @@
   desc: "Kong uses Nginx"
 - name: OpenResty
   link: http://openresty.org
-  desc: "OpenResty uses Nginx"
\ No newline at end of file
+  desc: "OpenResty uses Nginx"
+- name: sdsfdf
```
Comment (leecalcote) [inline _data/categories/gw.yml]: @madhabpaul, it looks like lines 16, 17 and 18 need to be blanked out.

Comment (madhabpaul) [reply _data/categories/gw.yml]: yaa @leecalcote  16, 17, 18 need to be removed

**Comment by leecalcote** [comment]
@madhabpaul I went ahead and added the Gemfile back.

File: _data/categories/gw.yml (hunk 1 above)
Comment (madhabpaul) [reply _data/categories/gw.yml]: I have removed the lines.

File: collections/_posts/_projects/meshery.md [hunk 2]
```diff
@@ -1,10 +0,0 @@
----
```
Comment (leecalcote) [inline collections/_posts/_projects/meshery.md]: Is the relocation of this file intentional?

File: _data/categories/gw.yml [hunk 3]
```diff
@@ -12,4 +12,22 @@
   desc: "Kong uses Nginx"
 - name: OpenResty
   link: http://openresty.org
-  desc: "OpenResty uses Nginx"
```
Comment (leecalcote) [inline _data/categories/gw.yml]: Nice work!

File: collections/_posts/_projects/meshery.md (hunk 2 above)
Comment (madhabpaul) [reply collections/_posts/_projects/meshery.md]: no

**Pull request merged**